src/passport.js

```diff
@@ -14,95 +14,100 @@ export default class Passport {
 
     app.set('storage', this.storage);
     this.getJWT().then(this.setJWT);
+
+    this.setupSocketListeners();
   }
 
-  connected () {
+  setupSocketListeners() {
```

Comment:
Now we only set up the reconnect listeners once per app instance instead of every time `connect` was called.